### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by Ilfan** [mgatelabs/SCN-VR#2]
- deleted -

**Comment by mgatelabs** [comment]
What exactly are you trying to do and some of the code is a little old, I have more changes but they have demo code that I need to remove.

**Comment by Ilfan** [comment]
Creating an iOS8+ app, it will prompt the user if they have Google cardboard yes/no, if no, it will switch to Mono video mode, if yes, it will be the SBS mode, now it slightly gets a little trickier, creating a playlist that will play automatically one after another that will have 5 videos, when one finishes the other will auto start, however, 3 out of the 5 videos will just need to be flat videos (no camera movement/no gyroscope mode/basically its just a flat panel), when the other 2 videos switch, i need them to be in a full sphere where the user can look around 360 mode and the video will be wrapping fully in 360 mode.

So basically what I need is the code to put the video on full sphere video, and if possible, the flat mode as well.

The user will not be able to change settings by themselves, so will need to preconfig that based on the yes/no option.

Much appreciated.

On Oct 17, 2015, at 5:49 PM, Michael Fuller <notifications@github.com<mailto:notifications@github.com>> wrote:

What exactly are you trying to do and some of the code is a little old, I have more changes but they have demo code that I need to remove.

**Comment by mgatelabs** [comment]
The one thing you need to deal with is the different cardboard and viewers.  That's why it has profiles which allow you to customize the experience.  I need to write some more later.
